inst/rmarkdown/templates/jasa/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: This is a title and this is too
+shorttitle: A subtitle goes on another line
+author:
+- name: Author's name
+  affiliation: Author's affiliation
+  email: e-mail\@uni.edu
+  thanks: other info
+- name: Second author's name
+  affiliation: Second author's affiliation
+output: rticles::jasa_article
+abstract: |
+  Put your abstract here. Abstracts are limited to 200 words for
+  regular articles and 100 words for Letters to the Editor. Please no
+  personal pronouns, also please do not use the words ``new'' and/or
+  ``novel'' in the abstract. An article usually includes an abstract, a
+  concise summary of the work covered at length in the main body of the
+  article.
+# The following prints the date (optional, set to false to remove).
+date: true
+---
+
+```{r setup, include=FALSE}
+# Use fig.process to rename figure files generated by R so that they comply with
+# the journal naming policies. Remember to name plot chunks with the format
+# "Figure1", "Figure2", etc... You will find the figure files in
+# `[...]_files/figure-latex/`.
+
+knitr::opts_chunk$set(echo = TRUE, fig.process = function(x) {
+  x2 = sub('-\\d+([.][a-z]+)$', '\\1', x)
+  if (file.rename(x, x2)) x2 else x
+})
+```
+
+Your article.
+
+```{r Figure1, include=TRUE, fig.cap="Your caption."}
+plot(pressure)
+```
+
+<!-- You can add your acknowledgements here (optional). -->
+::: {.acknowledgments data-latex=""}
+This research was supported by ...
+:::
+
+<!-- use \appendix* for only one appendix section -->
+\appendix 
+
+To start the appendix, use the `\appendix` command.
+This signals that all following section commands refer to appendixes
+instead of regular sections. Therefore, the `\appendix` command
+should be used only once---to set up the section commands to act as
+appendices. Thereafter normal section commands are used. The heading
+for a section can be left empty. For example,
+
+::: {.verbatim data-latex=""}
+\appendix
+\section{}
+:::
+
+will produce an appendix heading that says "APPENDIX A" and
+
+::: {.verbatim data-latex=""}
+\appendix
+\section{Background}
+:::
```

Comment:
BTW to show latex code in a Rmd file you can use 

````markdown
```latex
\appendix
\section{Background}
```
````

But it will be a code chunk with latex syntax highlighting and not the verbatim you code. 

It is just easier to write 😅